porcupine/plugins/filemanager.py

```diff
@@ -122,6 +157,28 @@ def rename(old_path: Path) -> None:
         tab.path = new_path / tab.path.relative_to(old_path)
 
 
+def paste(new_path: Path) -> None:
+    assert copy_path is not None
```

Comment:
Not needed for type checker to be happy, because this function doesn't use `copy_path`.

```suggestion
```